integration-tests/src/test/scala/za/co/absa/spline/JDBCWriteTest.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.coresparkadapterapi
+
+import java.sql.{DriverManager, ResultSet}
+
+import org.apache.spark.sql.types.{IntegerType, StringType, StructField, StructType}
+import org.apache.spark.sql.{Row, SaveMode}
+import org.scalactic.source.Position
+import org.scalatest._
+import za.co.absa.spline.common.TempDirectory
+import za.co.absa.spline.fixture.{AsyncSparkFixture, AsyncSplineFixture}
+import za.co.absa.spline.model.DataLineage
+import za.co.absa.spline.model.op.Write
+
+
+class JDBCWriteTest extends AsyncFlatSpec
+  with Matchers
+  with AsyncSparkFixture
+  with AsyncSplineFixture
+  with DerbyDatabase {
+
+  val tableName = "testTable"
+
+  //Returns dataframe with sama test data
```

Comment:
Please don't use comments unless really necessary. Instead try to use proper naming and code style to make the code self-explanatory.